src/CMakeLists.txt

```diff
@@ -754,6 +754,11 @@ function(upm_module_init)
     endif (BUILDSWIGNODE)
     if (BUILDSWIGJAVA)
       upm_swig_java()
+
```

Comment:
Can we put this under it's own 
if (BUILDSWIGCORDOVA) ...

For the dependency, we could check that the JAVA target exists:
if(TARGET javaupm_$libname}) ...